modules/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/artifact/value/ValueProviderExecutor.java

```diff
@@ -78,21 +83,17 @@ public ValueResult resolveValues(ParameterizedModel parameterizedModel,
           LOGGER.debug("Invoking connector's value provider: {} for component: {}", providerName,
                        parameterizedModel.getName());
         }
-        return resultFrom(withContextClassLoader(extensionClassLoader, () -> valueProviderMediator.getValues(providerName,
-                                                                                                             parameterValueResolver,
-                                                                                                             connectionSupplier(context),
-                                                                                                             configSupplier(context),
-                                                                                                             context
-                                                                                                                 .getConnectionProvider()
-                                                                                                                 .orElse(null)),
-                                                 ValueResolvingException.class,
+        return resultFrom(withContextClassLoader(extensionClassLoader,
+                                                 () -> getValues(providerName, valueProviderMediator,
+                                                                 parameterValueResolver, context),
+                                                 org.mule.sdk.api.values.ValueResolvingException.class,
                                                  e -> {
                                                    throw new ExecutorExceptionWrapper(e);
                                                  }));
       } finally {
         context.dispose();
       }
-    } catch (ValueResolvingException e) {
+    } catch (org.mule.sdk.api.values.ValueResolvingException e) {
```

Comment:
Is there any reference to the old ValueResolvingException? If not, change the import to the new one.